static/js/popovers.js

```diff
@@ -466,10 +466,10 @@ export function toggle_actions_popover(element, id) {
         let editability_menu_item;
         if (editability === message_edit.editability_types.FULL) {
             use_edit_icon = true;
-            editability_menu_item = $t({defaultMessage: "Edit"});
+            editability_menu_item = $t({defaultMessage: "Edit message"});
         } else if (editability === message_edit.editability_types.TOPIC_ONLY) {
             use_edit_icon = false;
-            editability_menu_item = $t({defaultMessage: "View source / Move message"});
+            editability_menu_item = $t({defaultMessage: "View message source / Move message"});
```

Comment:
I'd skip this in the first commit, since it's being replaced the second one.  (And likely is a broken intermediate state; it's likely too wide to fit).